test/createAzureResource.test.ts

```diff
@@ -50,22 +54,60 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
     });
 
     test('Create and Delete New Web App (Advanced)', async () => {
-        const resourceName: string = getRandomHexString().toLowerCase();
+        const resourceName2: string = getRandomHexString().toLowerCase();
         await vscode.workspace.getConfiguration(constants.extensionPrefix).update('advancedCreation', true, vscode.ConfigurationTarget.Global);
-        const testInputs: (string | RegExp)[] = [resourceName, '$(plus) Create new resource group', resourceName, 'Linux', regExpLTS, '$(plus) Create new App Service plan', resourceName, 'B1', 'West US'];
+        const testInputs: (string | RegExp)[] = [resourceName2, '$(plus) Create new resource group', resourceName2, 'Linux', regExpLTS, '$(plus) Create new App Service plan', resourceName2, 'B1', 'West US'];
         ext.ui = new TestUserInput(testInputs);
 
-        resourceGroupsToDelete.push(resourceName);
+        resourceGroupsToDelete.push(resourceName2);
         await vscode.commands.executeCommand('appService.CreateWebApp');
-        const client: WebSiteManagementClient = getWebsiteManagementClient(testAccount);
-        const createdApp: WebSiteManagementModels.Site = await client.webApps.get(resourceName, resourceName);
+        const createdApp: WebSiteManagementModels.Site = await webSiteClient.webApps.get(resourceName2, resourceName2);
         assert.ok(createdApp);
 
-        ext.ui = new TestUserInput([resourceName, DialogResponses.deleteResponse.title, DialogResponses.yes.title]);
+        ext.ui = new TestUserInput([resourceName2, DialogResponses.deleteResponse.title, DialogResponses.yes.title]);
         await vscode.commands.executeCommand('appService.Delete');
-        const deletedApp: WebSiteManagementModels.Site | undefined = await client.webApps.get(resourceName, resourceName);
+        const deletedApp: WebSiteManagementModels.Site | undefined = await webSiteClient.webApps.get(resourceName2, resourceName2);
         assert.ifError(deletedApp); // if app was deleted, get() returns null.  assert.ifError throws if the value passed is not null/undefined
     });
+
+    test('Create and Delete New Web App (Basic)', async () => {
```

Comment:
Hold off on this test for now.  We don't want to accidentally delete a user's default resource group if they happen to run tests locally (or if we accidentally run it and delete our tests).  

I think I will be adding a setting that allows users to change their default rg/asp name.  We can use that to test more safely :)